content/en/community/awards/2024/index.md

```diff
@@ -51,39 +51,41 @@ For boosting so many messages both to and about our contributors, and taking on
 
 #### SIG etcd
 
 *Ivan Valdes, [@ivanvc](https://github.com/ivanvc)*  
 Ivan has been extremely active across a number of etcd-io subprojects since getting involved with the project 12 months ago at our ContribFest in Chicago in November 2023 and has helped take sig-etcd tooling to another level. Ivan adding to your code & review contributions we are deeply appreciative of your focus on helping other community members and being active in our regular community and triage meetings, thank you for all your hard work. 
 
-*Siyuan Zhang, [@siyuanfoundation](https://github.com/siyuanfoundation)*
+*Siyuan Zhang, [@siyuanfoundation](https://github.com/siyuanfoundation)*  
 Siyuan has been making crucial contributions to some of our most important roadmap items for etcd, both within the etcd-io repository but also across a number of kubernetes subprojects relating to etcd. Siyuan the amount of progress you've contributed towards etcd 3.6 since your first pr just over a year ago is extraordinary. Thank you for your help towards making etcd 3.6 a reality and helping to bridge the divide between Kubernetes and etcd. Your efforts on livez/readyz, downgrades and etcd feature gates are making a huge difference.
 
-*Wei Fu, [@fuweid](https://github.com/fuweid)*
+*Wei Fu, [@fuweid](https://github.com/fuweid)*  
 Fu Wei has demonstrated extraordinary knowledge in crucial areas of etcd and bbolt. They were instrumental in landing etcd's extremely complex upgrade from grpc gateway from v1 to v2 and creating our new robustness testing suite for bbolt.
 
 #### Instrumentation
 
-*Manuel Rüger, [@mrueg](https://github.com/mrueg)*  
-Richa has made significant contributions to SIG Instrumentation, driving forward the Component SLI KEP as well as Kubelet Resource Metrics (not to mention her cross-SIG activity in API-Machinery via work on the MVP KEP).
-
 *Catherine Fang, [@catherineF-dev](https://github.com/CatherineF-dev)*  
 Catherine has easily been one of the most prominent contributors in Kube State Metrics since her involvement from the v2.6.0 release (August, 2022). Since then, she has not only unblocked us on various fronts by contributing crucial patches, but also been an excellent all-rounder, keeping the tooling and documentation in shape, as well as actively seeking out issues that affect the community, and addressing them timely. She has helped out with 204 pull request to date, while driving 34 pull requests of her own.
 
+*Manuel Rüger, [@mrueg](https://github.com/mrueg)*  
+Manuel has landed 144 pull requests to date, with patothes entailing major features, as well as keeping the release machineTV in shape which enabled him to deliver new releases ofn time.
+He has been active in the project since v1.5.0 (January, 2019), and been a major help ever since (involved in reviewing ~555 pull requests) Kube State Metrics could never have gotten to
```

Comment:
Kubernetes 1.5 was in 2016.  Is this supposed to say 2016, or is it supposed to say v1.14.0?

Reply:
I think SIG-Instrumentation leads here v1.5 means https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.5.0 😄.